File: the-descent.py
import sys
import math

# game loop
while True:
    index = 0
    height = 0

    for i in range(8):
        mountain_h = int(input())  # represents the height of one mountain.
        if (mountain_h > height):
            height = mountain_h
            index = i
    
    print(index) # the index of the mountain to fire on.

File: the-descent.swift
import Glibc
import Foundation

public struct StderrOutputStream: TextOutputStream {
  public mutating func write(_ string: String) { fputs(string, stderr) }
}
public var errStream = StderrOutputStream()

// game loop
while true {
  var index = 0
  var height = 0
  
  for i in 0...7 {
    let mountainH = Int(readLine()!)! // represents the height of one mountain.
      
    if mountainH > height {
      height = mountainH
      index = i
    }
  }

  print(index);     // The index of the mountain to fire on.
}
